src/lib/archive.js

```diff
@@ -14,6 +15,7 @@ module.exports = function (state, bus) {
     if (state.opts.http) serve(state, bus)
 
     if (state.writable && state.opts.import) doImport(state, bus)
+    else if (state.opts.sparse) selectiveSync(state, bus)
     else download(state, bus)
```

Comment:
It'd be cool to have a download progress UI =). I think we can update the dat-node/stats to work for this but that can wait.